chsdi/tests/integration/test_file_storage.py

```diff
@@ -93,6 +152,52 @@ def test_file_too_big_kml(self):
         resp = self.testapp.post('/files', data, headers=self.headers, status=413)
         self.assertIn('File size exceed', resp.body)
 
+    # Test if there are matches in KML with Regex:
+    # (\s+on\w*=(\"[^\"]+\"|\'[^\']+\'))
+    def test_kml_check_regex1(self):
+        # TEST: REGEX OCCURS ONCE
+        # First request, to get ids
+        resp = self.testapp.post('/files', INVALID_KML1_once, headers=self.headers, status=200)
+        file_id = resp.json['fileId']
+
+        # check kml based on invalid regex 1
+        resp = self.testapp.get('/files/%s' % file_id, headers=self.headers, status=200)
+        orig_data = resp.body
+        self.assertEqual(orig_data, VALID_KML)
+
+        # TEST: REGEX OCCURS SEVERAL TIMES
+        # First request, to get ids
+        resp = self.testapp.post('/files', INVALID_KML1, headers=self.headers, status=200)
```

Comment:
Why an invalid KML repond with 200?

Reply:
We decided to silently remove the probably malicious code and keep the rest.

We are doing the same in the front-end.

Reply:
ok got it, then just remove the prints in the services and add #NOQA as suggested and it's good to merge.